tensordict/_td.py

```diff
@@ -103,6 +102,11 @@
 except ImportError:  # torch 2.0
     from torch._dynamo import is_compiling as is_dynamo_compiling
 
+try:
+    from torch.nn.parameter import Buffer
```

Comment:
I didnt follow why this maybe needed. Havent `Buffer`s been available in PyTorch for quite some time ?

Reply:
Ohh interesting. Thank you for the link. I think I confused this with the `register_buffer` method.